cmd/incus-migrate/main_migrate.go

```diff
@@ -586,6 +590,12 @@ func (c *cmdMigrate) Run(cmd *cobra.Command, args []string) error {
 		}
 	} else {
 		_, ext, convCmd, _ := archive.DetectCompression(config.SourcePath)
+
+		_, err := exec.LookPath(convCmd[0])
```

Comment:
ConvCmd coupd be nil, leading to a panic, right?

Reply:
Oops, indeed. I fixed it now by moving it into the if statement below.

Reply:
@hallyn should be good to go now